helm/aladdin-demo/templates/aladdin-demo.service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata: 
+  name: {{ .Chart.Name }}
+  labels:
+    name: {{ .Chart.Name }}
+    app: {{ .Chart.Name }}
```

Comment:
Vinay and I added some new labels for each project - check latest websocket or mission control for them. We added them everywhere